locale/pt_BR/wivrn-dashboard.po

```diff
@@ -4,20 +4,21 @@
 # SPDX-FileCopyrightText: 2025 Claudio Sampaio (Patola)
 # LuckShiba <LuckShiba@protonmail.com>, 2025.
 #
+# SPDX-FileCopyrightText: 2025 myghi 63
 msgid ""
 msgstr ""
 "Project-Id-Version: WiVRn-dashboard\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2025-06-20 19:23-0300\n"
-"PO-Revision-Date: 2025-06-21 21:55-0300\n"
-"Last-Translator: LuckShiba <LuckShiba@protonmail.com>\n"
-"Language-Team: Brazilian Portuguese\n"
+"POT-Creation-Date: 2025-08-04 04:40-0300\n"
+"PO-Revision-Date: 2025-08-08 11:56-0300\n"
+"Last-Translator: myghi63 <myghi63@disroot.org>\n"
+"Language-Team: Portuguese <kde-i18n-pt_BR@kde.org>\n"
```

Comment:
Remove the `kde-i18n-pt_BR@kde.org`, we're not related to KDE (applies to wivrn.po as well)